fakesnow/server.py

```diff
@@ -28,8 +29,11 @@
 logger.handlers = logging.getLogger("uvicorn").handlers
 logger.setLevel(logging.INFO)
 
+
 shared_fs = FakeSnow()
 sessions: dict[str, FakeSnowflakeConnection] = {}
+# Global mapping for query results by sfqid
+sfqid_results: dict[str, dict] = {}
```

Comment:
i think probably the best place for this is in `instance.FakeSnow`, stored as a `dict[str, pyarrow.Table]` .. this would allow it to be used by the patching in a future PR